lib/spack/spack/test/cmd/config.py

```diff
@@ -38,6 +38,24 @@ def config_yaml_v015(mutable_config):
     return functools.partial(_create_config, data=old_data, section="config")
 
 
+def test_config_list_scopes():
+    output = config("list-scopes")
+    assert "command_line" in output
+    assert "_builtin" in output
+
+
+def test_config_list_scopes_file():
+    output = config("list-scopes", "--file")
+    assert "site" in output
+    assert "_builtin" not in output
+
+
+def test_config_list_scopes_non_platform():
+    output = config("list-scopes", "--non-platform")
+    assert "site" in output
+    assert "user" in output
```

Comment:
This test fails for me if `SPACK_DISABLE_LOCAL_CONFIG` is `true` when I run the tests.

Reply:
You might be able to get some inspiration from https://github.com/spack/spack/blob/8ac826cca892f9060ced53a071e78e0119dd8aca/lib/spack/spack/test/config.py#L1230.